client/src/Components/Reviews.jsx

```diff
@@ -8,16 +8,53 @@ class Reviews extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      load: false,
       mostPopular: [],
       allReviews: [],
     };
+    this.vote = this.vote.bind(this);
+  }
+
+  vote(e) {
+    e.persist();
+    this.setState({load: true}, () => {
+      const data = {
+        vote: e.target.name,
+        _id: e.target.className,
+      };
+      fetch('http://localhost:3002/reviews', {
+        method: 'PUT',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(data)
+      })
+        .then((review) => review.json())
+        // .then((review) => {
+        //   this.setState({reviews: results});
+        //   return results;
+        // })
+        .then((review) => {
+          this.setState({load: false});
+          return review;
+        })
+        .then((review) => console.log(review))
+        .catch((err) => console.log('There was an error'));
+    });
   }
 
   render() {
     return (
```

Comment:
This component is kind of big. I think it would be good to break it into sub-components.